internal/pkg/upgrade/upgrade.go

```diff
@@ -218,3 +219,15 @@ func upgradeBoot(url string) error {
 
 	return syslinux.Install(constants.BootMountPoint, bootcfg)
 }
+
+// Reset calls kubeadm reset to clean up a kubernetes installation
+func Reset() (err error) {
```

Comment:
Is it worth adding this to the current OSD implementation?

Reply:
I think we even have it today, but it's broken.

At the same time it's not possible to call OSD API from init today?

Reply:
Right. 🤔 This can probably wait until we land support for starting a `kubeadm reset` service.

Reply:
Yeah this is just a temp bandaid until we get the other bits sorted.